src/main/java/jdbchomework/utils/InjectorJdbc.java

```diff
@@ -45,10 +45,10 @@
     private static SkillController skillController = new SkillController(skillService);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)